llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -41,6 +41,7 @@ constexpr uint32_t SPIRV_PIPELINE_ENABLE_DECOR = 5919;
 enum class DecorValueTy {
   uint32,
   boolean,
+  string,
```

Comment:
Added a string DecorValueTy to enabling the store of a list of literals as a comma-separated string.